webpack/webpack.dev.js

```diff
@@ -0,0 +1,6 @@
+ const merge = require('webpack-merge');
```

Comment:
I do not think this package is necessary. A spread operator would suffice.

Reply:
Ok. We will implement that right away